### DISCUSSION TITLE
Unable to access Argocd UI

### DISCUSSION THREAD

**Discussion by maheshkoheda** [learndevopsonline/learndevopsonline#1636] [Q&A]
@r-devops Hi Raghu i have deployed manifests required and trying to access server for argocd UI. but not able to connect securely to redirected 127.0.0.1/8080. Help required here as i am not able to figure out what settings need to change i have tried allow all security group too but not able to connect further to work on CD part.
<img width="960" alt="Argocd UI not responding" src="https://github.com/learndevopsonline/learndevopsonline/assets/25144175/4926d53b-7494-4472-adaf-7bb135e11f20">

**Comment by sreedharm07** [comment]
hi @maheshkoheda 
did still issue exists ?

**Reply by maheshkoheda** [reply]
Yes @sreedharm07

**Reply by sreedharm07** [reply]
Anyway time is 9 14 now lab session is going on. please join there if possible

**Reply by maheshkoheda** [reply]
I tried to join now.. looks like session is finished.

**Reply by sreedharm07** [reply]
ok please install now and try if you still get same error please ping your ip here
